packages/victory-core/package.json

```diff
@@ -24,7 +24,7 @@
     "d3-scale": "^1.0.0",
     "d3-shape": "^1.2.0",
     "d3-timer": "^1.0.0",
-    "lodash": "^4.17.19",
+    "lodash": "^4.17.21",
```

Comment:
This change is only included so that the lerna will see a diff in victory-core and publish a new version of it and everything that depends on it (all victory packages) next time I publish.